src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
```

Comment:
The `add` method uses the arithmetic operator with the assignment operator as suggested by the [CODE STYLE] checklist point. This is a good use of the `+=` operator.